lib/ansible/plugins/cache/__init__.py

```diff
@@ -137,6 +137,14 @@ def validate_cache_connection(self):
                     raise AnsibleError("error in '%s' cache, configured path (%s) does not have necessary permissions (rwx), disabling plugin" % (
                         self.plugin_name, self._cache_dir))
 
+    def _get_cache_file_name(self, key):
+        prefix = self.get_option('_prefix')
+        if prefix:
+            cachefile = "%s/%s_%s" % (self._cache_dir, prefix, key)
```

Comment:
Does the addition of an underscore match the behaviour of RDBMS caches?

Reply:
Agree. To me the prefix should be just a prefix. If prefix is `xxx` for instance, it should be:

`xxxlocalhost`

Adding a prefix like `xxx_` will return `xxx_localhost` then